djqscsv/djqscsv.py

```diff
@@ -4,24 +4,32 @@
 
 from django.core.exceptions import ValidationError
 from django.utils.text import slugify
-from django.http import HttpResponse
+from django.http import HttpResponse, StreamingHttpResponse
```

Comment:
Yeah, I dropped `1.5` because it was failing on Travis CI when run with Python `2.6` (not sure why, logs were broken last night).

Since 1.6 and 1.7 are no longer supported, I dropped those as well, which let me get rid of some code too!